dashboard/templates/dashboard/blast_hist.html

```diff
@@ -1,53 +1,55 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <title>BLAST History</title>
-    <meta charset="utf-8">
-
-    <!-- META -->
-    <meta name="description" content="">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <!-- END META -->
-
-    <!-- FONTS -->
-    <link href='http://fonts.googleapis.com/css?family=Dosis:400,500,600' rel='stylesheet' type='text/css'>
-    <link href='http://fonts.googleapis.com/css?family=Abel' rel='stylesheet' type='text/css'>
-    <!-- END FONTS -->
+{% extends "app/layout.html" %}
+{% load staticfiles %}
+{% load pipeline %}
+{% load jsonify %}
 
+{% block head-scripts %}
     <!-- CSS -->
-    <link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
 
-    <link rel="stylesheet" type="text/css" href="static/dashboard/css/design.css">
+     <link rel="stylesheet" type="text/css" href="../static/dashboard/css/design.css">
```

Comment:
I think this line will not work on production environment, because the url will be something with `webapp/` prefix. I use this kind of syntax <link rel="stylesheet" type="text/css" href="{% static 'dashboard/css/design.css' %}">` to fix this issue in previous commits on this branch.

Reply:
@ifishlin Sorry, I didn't realize that my comment is somewhat broken because missing paired `. I will fix that by myself through few commits.